frontend/src/components/general-comparison-table/general-comparison-table.tsx

```diff
@@ -77,16 +85,34 @@ const GeneralComparisonTable: React.FC<{ isOnlyDiff: boolean }> = ({
     const broadcast = new BroadcastChannel('compare');
     broadcast.onmessage = (): void => {
       refetch();
+      refetchPrices();
     };
-  }, [refetch]);
-
-  if (isLoading) return <Spinner />;
+  }, [refetch, refetchPrices]);
+  if (isLoading && isLoadingPrices) return <Spinner />;
 
   return (
     <CollapseElement label="General information" isOpen={true}>
       <EqualHeight>
         <div className={clsx(styles.table, 'table')}>
           <div className={clsx(styles.tableTitles, styles.tableColumn)}>
+            {(!isOnlyDiff || !isIdentical.bodyType) && (
```

Comment:
Why do you use `!isIdentical.bodyType` here? It means when `bodyType` is the same for all cars this row will not be displayed. I'm pretty sure that's not what we need to achieve here

Reply:
You are right, it is mistake. Now it's corrected, we will show prices always.